components/merkledb/src/proof_list_index/mod.rs

```diff
@@ -823,3 +824,93 @@ where
         self.base_iter.next().map(|(_, v)| v)
     }
 }
+
+#[cfg(feature = "with-protobuf")]
+#[doc(hidden)]
+pub mod proto {
```

Comment:
Should this module be public? I am not sure, but it seems that trait implementations always public.